extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -810,7 +820,10 @@ export class AmpA4A extends AMP.BaseElement {
         return fetchResponse;
       })
       .then((fetchResponse) =>
-        this.startValidationFlow_(fetchResponse, checkStillCurrent)
+        getExperimentBranch(this.win, NO_SIGNING_EXP.id) ===
+        NO_SIGNING_EXP.control
+          ? this.streamResponse_(fetchResponse)
```

Comment:
should it be the reverse?

Reply:
Yikes, good catch. No idea why the tests passed.

Reply:
Oh I think its because this returns `undefined`

Reply:
ye, actually should change control to experiment, as @powerivq pointed out in the other comment.

Reply:
done.